### ISSUE TITLE
user-guide: add info about --no-scm mode (and link from get-started)

### ISSUE THREAD

**Issue by efiop** [iterative/dvc.org#100]
Maybe worth making some portions of current get-started optional. ~~Especially with the upcoming Hg support.~~

UPDATE: Skip to https://github.com/iterative/dvc.org/issues/100#issuecomment-657783092 for more details.

**Comment by shcheklein** [comment]
Should be part of the user-guide. A link in the get started should enough. It's not a prevalent use case to distract attention on it.

**Comment by algomaster99** [comment]
@shcheklein Is this issue about `dvc init --no-scm` command? How does it change the workflow? For example, `dvc diff` wouldn't work since the project won't have any commits.

**Comment by shcheklein** [comment]
@algomaster99 yes, some commands won work. I would say, it a very low priority issue. Let's not focus on this one for now.

**Comment by dashohoxha** [comment]
Working on a Katacoda scenario for this: https://katacoda.com/dvc/scenarios/data-management

**Comment by dashohoxha** [comment]
I have created a lesson/tutorial about using DVC in `--no-scm` mode: https://katacoda.com/dvc/courses/basics/data
Is this sufficient to close the issue?

**Comment by shcheklein** [comment]
@dashohoxha not enough, I believe. I think this ticked should be implemented as a note/mention in one of the introductory user guide sections (that are still not implemented and probably will be taken from the Understanding DVC). I can imagine there can be a section named "How it works" that can have a page inside "DVC and Git" - just an example to give you some sense. We should come with better titles for this. 

Bottom line, I believe it'll be closed as part of #425, and #144, and #550.

**Comment by jorgeorpinel** [comment]
Do we still want to note this in Get Started? Seems to me we no longer want to do this, in order to keep the start guide as lean and non-distracting as possible. The option is explained in https://dvc.org/doc/command-reference/init and mentioned in a few other refs already (e.g. in add and config) so should we close this?

**Comment by efiop** [comment]
@jorgeorpinel We do have the cmdreference, but we need some user-guide article about it that will describe what and how it could be used for(e.g. docker, deployment etc). I wouldn't close it for now.

**Comment by jorgeorpinel** [comment]
Maybe a "How To use init --no-scm effectively"? Rel #899

**Comment by efiop** [comment]
@jorgeorpinel I don't quite understand what is `effectiveness` in `--no-scm` though? how-tos seem like simple recipes to me, so in that particular format they would be a multiple of articles (e.g. using data in docker). That's a great idea, but i feel like it also needs a user-guide article to describe what it is about and how it works and then maybe reference all the how-tos you could use --no-scm in.

**Comment by jorgeorpinel** [comment]
The point was it could be a How To guide (the title can be anything), since we have that ticket already.

> but i feel like it also needs a user-guide article to describe what it is about and how it works and then maybe reference all the how-tos

Fair enough.

**Comment by shcheklein** [comment]
Closing this. We def would like to add articles like "Packaging DVC artifacts into Docker", "DVC in production", etc ... but we have info regarding `--no-scm` itself. I don't think we want a whole separate section on it.
